### ISSUE TITLE
cargo install jless

### ISSUE THREAD

**Issue by tox2ik** [PaulJuliusMartinez/jless#120]
for some (crazy) reason this is not mentioned as a way to install in the main readme.
brew is mentioned, but cargo is not.
is it because it's not a binary package manager? :shrug:

imho it should not be necessary to spend 2+ minutes looking for a "copy paste" solution / instructions to install.
i almost wound up setting up brew. and im on gentoo.

just saying :) 

maybe include "cargo install jless" in the readme.

cheers!
works great.

(not crazy about the dimming effect when keying up and down, but hey it's probably there for a reason?)

**Comment by GeroVanMi** [comment]
`cargo install` is mentioned in the main README. (Maybe this was changed since this was posted.) 
However, it wasn't in the table with all the other installation methods. I've created a pull request where I moved it into the table and added a link to the Rust Toolchain installation. 

Preview from my fork:
![image](https://github.com/PaulJuliusMartinez/jless/assets/39983753/494d4adf-1332-4103-94ee-aec15dcbc858)

@PaulJuliusMartinez
